compiler/semexprs.nim

```diff
@@ -2284,11 +2284,16 @@ proc semMagic(c: PContext, n: PNode, s: PSym, flags: TExprFlags): PNode =
     markUsed(c, n.info, s)
     when defined(leanCompiler):
       localError(c.config, n.info, "compiler was built without 'spawn' support")
-      result = n
+      result = errorNode(c, n)
     else:
       result = setMs(n, s)
       for i in 1..<n.len:
         result[i] = semExpr(c, n[i])
+
+      if n.len > 1 and n[1].kind notin nkCallKinds:
+        localError(c.config, n[1].info, "'spawn' takes a call expression; got " & $n[1])
```

Comment:
```suggestion
        localError(c.config, n[1].info, "'spawn' takes a call expression; got: " & $n[1])
```
Or something else, so that the expression and the message are separated.

Reply:
Yeah, I will change, it's pre-existing(I copied the messages)